lib/packs/private.rb

```diff
@@ -206,6 +203,12 @@ def self.move_to_parent!(
           end
         end
 
+        if other_package.name == parent_name &&
```

Comment:
This had to be moved after replacing ignored dependencies so we can look for the new package name